src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ *
+ * ClaimForm
+ *
+ */
+
+import React, { useMemo } from 'react';
+import cn from 'classnames';
+import { useTranslation, Trans } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import { Input } from 'app/components/Form/Input';
+import { AssetRenderer } from 'app/components/AssetRenderer';
+import { Asset } from 'types';
+import { Button } from 'app/components/Button';
+import { useSendContractTx } from '../../../../hooks/useSendContractTx';
+import { TxType } from 'store/global/transactions-store/types';
+import { useCacheCallWithValue } from 'app/hooks/useCacheCallWithValue';
+import { TxDialog } from 'app/components/Dialogs/TxDialog';
+import { weiToNumberFormat } from '../../../../../utils/display-text/format';
+import { useMaintenance } from 'app/hooks/useMaintenance';
+import { ErrorBadge } from 'app/components/Form/ErrorBadge';
+import { discordInvite } from 'utils/classifiers';
+
+interface Props {
+  className?: object;
```

Comment:
Why is this `object`? Usually, it's `string`.

Reply:
Oh, copied component without checking on it :|

Reply:
No problem :)